src/sentry/rules/processor.py

```diff
@@ -249,7 +257,56 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 else:
                     self.grouped_futures[key][1].append(rule_future)
 
-    def apply(self) -> Iterable[Any]:
+    def _get_active_release_rule_actions(self) -> Sequence[EventAction]:
+        # TODO: we need this to be configurable on a pre-project level?
+        return [
+            NotifyActiveReleaseEmailAction(
+                project=self.project, data={"targetType": ActionTargetType.RELEASE_MEMBERS.value}
+            )
+        ]
+
+    def apply_active_release_rule(
```

Comment:
The structure of this method follows the `apply` method above. It just removes any configurable dynamism since that flexibility isn't needed.